draft-ietf-scitt-architecture.md

```diff
@@ -817,72 +818,234 @@ The last two steps may be shared between a batch of Signed Statements recorded i
   * tstr => any
 }
+
+Receipt_Protected_Header = {
+    -111 => int               ; Type of Verifiable Data Structure,
+                              ; for example "RFC9162_SHA256", or "CCF"
+
+    2 => [+ label]            ; Critical headers
+    13  => Receipt_CWT_Claims ; CBOR Web Token Claims
+    390 => int                ; SCITT Receipt Version
+
+    ? 4 => bstr        ; Key ID (optional)
+    ? 33 => COSE_X509  ; X.509 chain (optional)
+    ? 393 => Reg_info  ; Registration policy information (optional in Receipts)
+}
+
+Receipt_Headers = (
+    protected : *serialized* Receipt_Protected_Header,
+    unprotected : Receipt_Unprotected_Header
+)
+
+Receipt_as_COSE_Sign1 = [
+    Receipt_Headers,
+    payload : bstr / nil,
+    signature : bstr
+]
+
+Receipt = #6.18(Receipt_as_COSE_Sign1)
+
+; A Transparent Statement is a Signed Statement
+; with one or more Receipts in it's unprotected header.
+
+Signed_Statement_CWT_Claims = {
+  1 => tstr; iss, the issuer
+  1 => tstr; sub, the subject
+  * tstr => any
+}
+
+Signed_Statement_Headers = {
+    1 => int                            ; Signing Algorithm
+    13  => Signed_Statement_CWT_Claims  ; CBOR Web Token Claims
+    ? 4 => bstr         ; Key ID (optional)
+    ? 33 => COSE_X509   ; X.509 chain (optional)
+    393 => Reg_info     ; Registration policy information (mandatory in Signed_Statement)
+}
+
+Receipt_Unprotected_Header = {
+  &(receipts: -333) => [+ Receipt]
+}
+
+Signed_Statement_Headers = (
+    protected : *serialized* Signed_Statement_Protected_Header,
+    unprotected : Signed_Statement_Unprotected_Header
+)
+
+Transparent_Signed_Statement_as_COSE_Sign1 = [
+    Signed_Statement_Headers,
+    payload : bstr / nil,
+    signature : bstr
+]
+
+Transparent_Signed_Statement = #6.18(Transparent_Signed_Statement_as_COSE_Sign1)
+
 ~~~
 
+
+Here is an example transparent statement:
+
+### Example
+
+#### Signed Statement
+
+~~~~ cbor-diag
+18(                                 / COSE Sign 1                   /
+    [
+      h'a4012603...6d706c65',       / Protected                     /
+      {                             / Unprotected                   /
+        -333: [                     / Receipts (1)                  /
+          h'd284586c...4191f9d2'    / Receipt 1                     /
+        ]
+      },
+      h'',                          / Detached payload              /
+      h'79ada558...3a28bae4'        / Signature                     /
+    ]
+)
+~~~~
+
+Notice the payload is detached,
+this is to support very large supply chain artifacts, and to
+ensure that Transparent Statements can integrate with
+existing file systems.
+
+Notice the unprotected header can contain multiple receipts.
+
+#### Signed Statement Protected Header
+
+~~~~ cbor-diag
+{                                   / Protected                     /
+  1: -7,                            / Algorithm                     /
+  3: application/example+json,      / Content type                  /
```

Comment:
if we dare, we can make this an actual well known software supply chain media type

Reply:
yes, we should... what that type should be, that's another fun conversation.

Reply:
Let's not block this PR on this. Picking one of the BOM types could bias the architecture and we want to be completely neutral